source/fundamentals/logging.txt

```diff
@@ -0,0 +1,285 @@
+
+Configure Your Logger
+---------------------
+
+To configure your logger, you must configure the logging framework bound to
+SLF4J.
+
+The following subsection provides a concrete example showing how to configure
+your logger.
+
+Configure Your Log Level
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this subsection we show how to configure your logger's **log level**.
+
+A logger's log level specifies a lower bound for how urgent a message must be
+for the logger to output it.
+
+This example shows how to configure your logger's log level to INFO.
+Select the tab corresponding to the logging framework you are using in your
+project.
+
+.. tabs::
+
+   .. tab:: Logback
+      :tabid: logback-binding
+
+      Specify Logback configurations in a file named ``logback.xml``. Your
+      ``logback.xml`` file does not have to be in a specific location, but it must
+      be accessible from your classpath. 
+
+      The Logback framework defines the following log levels. You can
+      see the log levels listed below in order from most urgent to least
+      urgent: 
+      
+      - ERROR
+      - WARN
+      - INFO
+      - DEBUG
+      - TRACE 
+
+      To configure your Logback logger to log all messages of level INFO or
+      higher, set your ``logback.xml`` file to the following.
+
+      .. code-block:: xml
+
+         <configuration>
+         <appender name="CONSOLE"
+           class="ch.qos.logback.core.ConsoleAppender">
+           <!-- deny all events with a level below INFO, that is TRACE and DEBUG -->
+           <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+             <level>INFO</level>
+           </filter>
+           <encoder>
+             <pattern>
+               %-4relative [%thread] %-5level %logger{30} - %msg%n
+             </pattern>
+           </encoder>
+         </appender>
+         <root level="DEBUG">
+           <appender-ref ref="CONSOLE" />
+         </root>
+         </configuration>
+
+      The above ``logback.xml`` file is a copy of the
+      `Threshold Filter Example in chapter seven of the Logback Manual <https://logback.qos.ch/manual/filters.html#thresholdFilter>`__.
+
+      For more information on configuring Logback, see the 
+      `the Logback Manual <https://logback.qos.ch/manual/configuration.html>`__.
+
+      .. tabs::
+
+   .. tab:: Log4j2
+      :tabid: log4j2-binding
+
+      Specify Log4j2 configurations in a file named ``log4j2.xml``. Your
+      ``log4j2.xml`` file does not have to be in a specific location, but it must
+      be accessible from your classpath. 
+
+      The Log4j2 framework defines the following log levels. You can see the
+      log levels listed below in order from most urgent to least urgent: 
+      
+      - OFF
+      - FATAL
+      - ERROR
+      - WARN
+      - INFO
+      - DEBUG
+      - TRACE
+      - ALL 
+
+      To configure your Log4j2 logger to log all messages of level INFO or
+      higher, set your ``log4j2.xml`` file to the following.
```

Comment:
Again, repetitive.

Reply:
Thanks for the note. Response to equivalent comment earlier: https://github.com/mongodb/docs-java/pull/76#discussion_r621257383